k8s/resources/study/config/study-server-application.yml

```diff
@@ -1,2 +1,14 @@
 non-evacuated-energy:
   default-provider: OpenLoadFlow
+
+# 2 servers are optional (not delivered in open-source)
+gridsuite:
+  services:
+    -
+      name: state-estimation-server
+      base-uri: http://state-estimation-server/
+      optional: true
```

Comment:
It's a private component, our guidline is to not have any reference to private components in public repositories.

Reply:
yes I took the easy way. did not see how include this in gridsuite-deployement